Sources/Commands/SwiftTool.swift

```diff
@@ -85,7 +85,7 @@ private class ToolWorkspaceDelegate: WorkspaceDelegate {
     private let stdoutStream: OutputByteStream
 
     init(_ stdoutStream: OutputByteStream) {
-        self.stdoutStream = stdoutStream
+        self.stdoutStream = ThreadSafeOutputByteStream(stdoutStream)
```

Comment:
I think stdoutStream and stderrStream should always be an instance of `ThreadSafeOutputByteStream`

Reply:
But `ToolWorkspaceDelegate` can't enforce that stdout or stderr will be passed to it. This way, whatever stream is passed, we know we've surrounded it with a thread-safe wrapper. What do you think?

Reply:
ah, right.